WordPress/Docs/PHP/PregQuoteDelimiterStandard.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0"?>
+<documentation xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:noNamespaceSchemaLocation="https://phpcsstandards.github.io/PHPCSDevTools/phpcsdocs.xsd"
+    title="Preg Quote Delimiter"
+    >
+    <standard>
+    <![CDATA[
+    Passing the $delimiter parameter to `preg_quote()` is strongly recommended to ensure the regular expression delimiter is properly escaped if it occurs in the input string.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: The delimiter parameter is passed.">
+        <![CDATA[
+$quoted_input = preg_quote( $input, <em>'#'</em> );
+preg_match(
+    <em>'#^' . $quoted_input . '#i'</em>,
+    $post_content,
+    $matches
+);
+        ]]>
+        </code>
+        <code title="Invalid: The delimiter parameter is missing.">
+        <![CDATA[
+$quoted_input = preg_quote( $input<em></em> );
```

Comment:
Would this make it even more clear ?

```suggestion
// $input = 'my #1 fan!';
$quoted_input = preg_quote( $input<em></em> );
```

Reply:
Ah, but now the examples look unbalanced. If the above is added, it should be added to both sides of the comparison.

Reply:
Done, I added the same comment to the valid example. I incorrectly assumed that you preferred adding it just to the invalid example for some reason.

Reply:
Thanks. I'll try to spell these things out more going forward.